extensions/yaml_support/labgraph_monitor/server/serializer_node.py

```diff
@@ -1,37 +1,120 @@
 #!/usr/bin/env python3
-# Copyright 2004-present Facebook. All Rights Reserve
+# Copyright 2004-present Facebook. All Rights Reserved.
 
+from dataclasses import field
+from typing import Dict, Optional
 import labgraph as lg
 import asyncio
 from labgraph.websockets.ws_server.ws_server_stream_message import (
     WSStreamMessage
 )
 from ..aliases.aliases import SerializedGraph
 
+# Make it work with RandomMessage
+from ....graphviz_support.graphviz_support.tests.demo_graph.random_message import RandomMessage
 
 class SerializerConfig(lg.Config):
     data: SerializedGraph
+    sub_pub_match: Optional[Dict] = field(default_factory=dict)
     sample_rate: int
     stream_name: str
     stream_id: str
 
+class DataState(lg.State):
+    data_1: Optional[Dict] = None
+    data_2: Optional[Dict] = None
+    data_3: Optional[Dict] = None
+    data_4: Optional[Dict] = None
 
 class Serializer(lg.Node):
```

Comment:
Is Serializer a generic class? There seems to be some hard-coded parts, e.g. input topics in this class.